src/app/Upload.svelte

```diff
@@ -113,4 +123,10 @@
   <span>or</span>
   <label for={UPLOAD_INPUT_ID} class="upload-btn">Browse file</label>
   <input id={UPLOAD_INPUT_ID} type="file" on:change={handleChange} />
+  <a
+    href={downloadURL ? downloadURL : "#"}
+    download="Example.kmp"
+    class="download"
```

Comment:
```suggestion
    class="download-link"
```
**[Suggestion]**: It's great to have more explicit class name, which it will later alleviate your testing

Reply:
I agree, `.download-link` is a better name, but never use a CSS class name in testing! 💀 😭 😭

Reply:
I am aware that some of the code is using id for testing and I thought this is what we have landed as a team, which is also why I put this comment here.